packages/fx-core/src/plugins/resource/bot/progressBars.ts

```diff
@@ -2,29 +2,24 @@
 // Licensed under the MIT license.
 
 import { IProgressHandler, PluginContext, v2 } from "@microsoft/teamsfx-api";
+import { Context } from "@microsoft/teamsfx-api/build/v2";
 
 export class ProgressBarFactory {
-  // To store working progress bars.
+  // To store working in progress bars.
   private static progressBars: Map<string, IProgressHandler | undefined> = new Map();
 
   public static async newProgressBar(
     name: string,
     steps_num: number,
     context: PluginContext | v2.Context
-  ): Promise<IProgressHandler | undefined> {
```

Comment:
As we discussed previously, I suggest not breaking the original design. If you're confident enough, please ignore this.

Reply:
ok, done it